CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # bedrock-tokenization ChangeLog
 
+## 5.0.0 - 2021-03-xx
+
+### Added
+- **BREAKING**: Implement `levelOfAssurance` in resolve endpoint.
+- **BREAKING**: Implement `minAssuranceForResolution` in create endpoint.
```

Comment:
I assume this is breaking because these are now `required` fields.  If that's the case, please update the verbiage to something like:  `Implement new required field `levelOfAssurance` in resolve endpoint.`